02-technical-guides/hierarchies/02-hierarchy-manipulations.ipynb

```diff
@@ -0,0 +1,2036 @@
+{
```

Comment:
<p>To be consistent, can we change the comment here to specifically say "alias" instead of the "set variables" language.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/atoti/atoti/pull/890/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>This is done, thanks for catching this!</p>